docs/excel/excel-add-ins-advanced-concepts.md

```diff
@@ -1,14 +1,13 @@
 ---
-title: Excel JavaScript API advanced concepts
+title: Advanced programming concepts with the Excel JavaScript API
 description: ''
-ms.date: 1/18/2018
+ms.date: 10/03/2018
 ---
 
 
+# Advanced programming concepts with the Excel JavaScript API
 
-# Excel JavaScript API advanced concepts
-
-This article builds upon the information in [Excel JavaScript API core concepts](excel-add-ins-core-concepts.md) to describe some of the more advanced concepts that are essential to building complex add-ins for Excel 2016 or later.
+This article builds upon the information in [Fundamental Excel add-in programming](excel-add-ins-core-concepts.md) to describe some of the more advanced concepts that are essential to building complex add-ins for Excel 2016 or later.
```

Comment:
link text should be changed to:  Fundamental programming concepts with the Excel JavaScript API